modules/tinymce/CHANGELOG.md

```diff
@@ -36,14 +36,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - When scrolling, the context toolbar will stick to where it was previously for large elements, such as tables #TINY-7545
 - The context toolbar will now move out of the way when it overlaps with the selection, such as in table cells #TINY-7192
 - The context toolbar will now use transition animations when changing placements #TINY-7740
-- The `formatter.match` API can now take an optional `similar` parameter to check if the format partial matches #TINY-7712
+- The `formatter.match` API can now take an optional `similar` parameter to check if the format partially matches #TINY-7712
 - `Env.browser` now uses the User-Agent Client Hints API when it is available #TINY-7785
 - Icons with a `-rtl` suffix in their name will now automatically be used when the UI is rendered in right-to-left mode #TINY-7782
 - `editor.formatter.formatChanged` now supports listening for changes to formats with specific variables #TINY-7713
 - The `autolink` plugin link detection now permits custom protocols and improves valid link detection #TINY-7714
 
 ### Changed
-- Changed the load order so that the content css gets loaded before the editor gets populated with contents #TINY-7249
+- Changed the load order so that the content CSS is loaded before the editor gets populated with contents #TINY-7249
```

Comment:
```suggestion
- Changed the load order so that the content CSS is loaded before the editor gets populated with content #TINY-7249
```
Should it be just content singular?